lbrynet/core/BlobManager.py

```diff
@@ -226,6 +226,14 @@ def _get_all_blob_hashes(self):
         return d
 
     @rerun_if_locked
+    @defer.inlineCallbacks
+    def _get_all_should_announce_blob_hashes(self):
```

Comment:
I think that for the tests you should mock time.time() and use the real `_get_blobs_to_announce` call, since this function isn't the same as the one that's used for real.